daemon/cmd/endpoint.go

```diff
@@ -456,6 +457,15 @@ func (d *Daemon) createEndpoint(ctx context.Context, owner regeneration.Owner, e
 				}).Warningf("Endpoint has %s annotation, but BPF bandwidth manager is disabled. This annotation is ignored.",
 					bandwidth.EgressBandwidth)
 			}
+			if hwAddr, ok := annotations["cni.cilium.io/macAddr"]; ok {
```

Comment:
Can we add the annotation in [here](https://github.com/cilium/cilium/blob/0f6f8980949276327fd77305d3ee2e05c268d573/pkg/annotation/k8s.go) and use a `const` instead?